src/PowerShell/Microsoft.WinGet.Client/Module/Microsoft.WinGet.Client.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package >
+  <metadata>
+    <id>Microsoft.WinGet.Client</id>

```

Comment:
Is this nuget going to nuget.org? Our reserved name is Microsoft.WindowsPackageManager.*